src/mumble/AudioOutput.cpp

```diff
@@ -404,6 +404,21 @@ void AudioOutput::initializeMixer(const unsigned int *chanmasks, bool forceheadp
 	}
 }
 
+void AudioOutput::prepareOutputBuffers(unsigned int frameCount, QList< AudioOutputBuffer * > &qlMix,
+									   QList< AudioOutputBuffer * > &qlDel) {
+	// Get the users that are currently talking (and are thus serving as an audio source)
+	QMultiHash< const ClientUser *, AudioOutputBuffer * >::const_iterator it = qmOutputs.constBegin();
+	for (int i = 0; i < qmOutputs.count(); i++) {
+		AudioOutputBuffer *buffer = it.value();
```

Comment:
```suggestion
		const AudioOutputBuffer *buffer = it.value();
```

Reply:
Compiler complains if I do this :P because of the virtual call to prepareBuffer

Reply:
Ah right - that function is not `const` and thus `buffer` can't be either

Reply:
Yep! :D